extension/src/prototypes/view/selection/ColorSchemeContent.tsx

```diff
@@ -81,6 +91,13 @@ const QualitativeContent: FC<{
         onClose={onClose}
       />
       <Divider />
+      <Button
+        variant="outlined"
+        size="small"
+        style={{ float: "right", marginRight: "10px", marginTop: "8px" }}
+        onClick={handleColorReset}>
+        Color Reset
```

Comment:
Should this be translated?
```suggestion
        色をリセット
```